### ISSUE TITLE
Improved Neural Network Archs

### ISSUE THREAD

**Issue by JonasRSV** [JonasRSV/Friday#3]
Current arch struggles to get a loss below 50/60, for validation & train. Overfitting has not been a problem so far, the problem lies in finding an arch with low enough latency that can learn the problem better.

![TensorBoard — Mozilla Firefox_048](https://user-images.githubusercontent.com/9198978/118391150-1a8d7e80-b633-11eb-86da-b1667f6e80e7.png)

Although this model and even models with a loss of around 150 works pretty well in a real  setting, I believe this loss correlates strongly with the false-positive rate of the model. By increasing the distance (reducing the loss) of negative samples it'll likely give a more stable behavior.

Current best arch is [this one](https://github.com/JonasRSV/Friday/blob/main/mm/models/bulbasaur/architechtures/kaggle.py#L58)

For more information on training models / getting data see [mm](https://github.com/JonasRSV/Friday/tree/main/mm)
